fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -198,23 +237,87 @@ private fun NormalTabsTabIcon(normalTabCount: Int) {
     }
 }
 
+/**
+ * Banner displayed in multi select mode.
+ *
+ * @param bannerItems List of icons to be displayed in the banner.
+ * @param menuItems List of menu items to be displayed in the menu at the end of the banner. Null if
+ * menu should not be displayed.
+ */
 @Composable
-private fun MultiSelectBanner() {
-    Box(
+private fun MultiSelectBanner(
+    selectedTabs: Int,
+    bannerItems: List<BannerItem> = emptyList(),
+    menuItems: List<MenuItem>? = null,
+) {
+    var showMenu by remember { mutableStateOf(false) }
+
+    Row(
         modifier = Modifier
             .fillMaxWidth()
-            .height(80.dp)
+            .height(88.dp)
             .background(color = FirefoxTheme.colors.layerAccent),
-        contentAlignment = Alignment.Center,
     ) {
+        IconButton(
+            onClick = {},
+            modifier = Modifier.align(Alignment.CenterVertically),
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_close),
+                contentDescription = stringResource(id = R.string.tab_tray_close_multiselect_content_description),
+                tint = FirefoxTheme.colors.iconOnColor,
+            )
+        }
         Text(
-            text = "Multi selection mode",
-            color = FirefoxTheme.colors.textOnColorPrimary,
+            text = stringResource(R.string.tab_tray_multi_select_title, selectedTabs),
```

Comment:
The `style` modifier currently [applied](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/layout/component_tabstray2.xml#60) to the XML version text does not affect the text style. The text style used in this PR should reflect the XML version using the `android:textStyle` attribute instead.

Reply:
Could you file a follow-up to check with UX what the text style should be today? I'll add it to the meta.